src/components/Inbox/NewMessage.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useSession } from '@inrupt/solid-ui-react';
+import { sendMessageTTL } from '../../utils';
+
+/**
+ * New Message Component - Component that allows user to write
+ *  a message to another user from their inbox
+ *
+ * @memberof Inbox
+ * @name NewMessage
+ */
+
+const NewMessage = () => {
+  const { session } = useSession();
+  const [message, setMessage] = useState({
+    recipientUsername: '',
+    title: '',
+    message: ''
+  });
+  const [error, setError] = useState('');
+  const [success, setSuccess] = useState('');
+
+  // Modifies message upon input
+  const handleChange = (e) => {
+    setMessage({
+      ...message,
+      [e.target.id]: e.target.value
+    });
+  };
+
+  // Handles submit (awaiting functionality for this)
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    if (!message.title) {
+      setError('Please enter a value for Message Title');
+    } else if (!message.recipientUsername) {
+      setError('Please enter a value for Recipient Username');
+    } else if (!message.message) {
+      setError('Please enter a value for the Message');
+    } else {
+      try {
+        await sendMessageTTL(session, message);
+        setMessage({
+          recipientUsername: '',
+          title: '',
+          message: ''
+        });
+        setError('');
+        setSuccess(`Message successfully sent to ${message.recipientUsername}`);
+      } catch (err) {
+        // TODO: Make sure invalid username is the only possible error
+        setError('Recipient username does not exist');
```

Comment:
"Recipient username not found" is more accurate

Reply:
Fixed!